src/api/requests.js

```diff
@@ -97,4 +57,90 @@ export const fetchValidPoints = async (levelId, groupId) => {
         }
     })
     return features
+}
+
+export const publishCatchment = async (body) => {
+    try {
+        const features = await getCatchmentGeoJSON(body.id)
+        const orgUnits = await ky.get(`${baseURL}/organisationUnits.json?fields=id,displayName~rename(name)&paging=false`, options).json()
+        // this endpoint posts an attribute and returns uid
+        const resp = await ky.post(`${baseURL}/attributes`, { body: JSON.stringify(body.payload), headers: options }).json()
+
+        // use this id to store with the catchment areas
+        const attributeId = resp?.response?.uid
+    
+        options["Content-Type"] = "application/json-patch+json"
+
+        for (let i=0; i<features.length; i++) {
+            // get the org unit and check to see that it exists in DHIS2
+            const orgId = features[i].properties["user:orgUnitId"]
+            const exists = orgUnits.organisationUnits.find((unit) => unit.id === orgId)
+
+            if (exists !== undefined) {
```

Comment:
What happens if it doesn't exist?

Reply:
if the org unit doesn't exist then it won't add a geojson since there's nothing to add it to.